### ISSUE TITLE
Display Time only as Positive Values

### ISSUE THREAD

**Issue by mboneil10** [umts/pvta-multiplatform#453]
In development mode, I ran into the following issue:

<img width="907" alt="screen shot 2018-03-12 at 12 50 11 pm" src="https://user-images.githubusercontent.com/19267771/37297547-5816365c-25f4-11e8-966b-837e319c1dbb.png">

Should probably say "early" or "on time" next to the clock symbol. Not sure if this is also happening in production.

**Comment by dfaulken** [comment]
They're two separate data points, right? The status of the bus and its EDT? Arguably we're just accurately reporting inconsistent data.

**Comment by werebus** [comment]
I'm not 100% sure, but I believe @akaplo found that the [only two values ever reported were "Late" and "On Time"][1]. Indeed that is the case e.g. right now:

```
$ curl -s --header 'Accept: application/json' 'https://bustracker.pvta.com/InfoPoint/rest/Vehicles/GetAllVehicles' | jq '.[] | .DisplayStatus' | sort -u
"Late"
"On Time"

$ curl -s --header 'Accept: application/json' 'https://bustracker.pvta.com/InfoPoint/rest/Vehicles/GetAllVehicles' | jq '.[] | .Deviation' | sort -n
-5
-5
-4
-2
0
#...
0
1
#...
6
7
9
9
10
12
20
```

[1]:  https://github.com/umts/pvta-multiplatform/blob/b7664c61d1e07347e600ce16f203a355616ef73c/src/pages/route/route.html#L63-L71

**Comment by werebus** [comment]
Ugh. At first I thought, "Oh, look! there's a field `OpStatus` that sometimes contains the string "EARLY", perhaps that's a good source of information." Nope. Take a look at [this grab-bag of :cow:](https://gist.github.com/werebus/9b504b9c53f92e5868d93e4348518d30)

**Comment by akaplo** [comment]
Right, a vehicle's `DisplayStatus` is either `On Time` or `Late`. Unfortunately, best approach for our users is probably to check for a negative `Deviation`, flip `DisplayStatus` to whichever it _isn't_ equal to, and strip out the negative. Ugh!

**Comment by mboneil10** [comment]
I think with this one @werebus, @sherson and I decided that it would be best to ask Avail if we can display "Early" or strip out the negative without any bad side effects.
